notebooks/introduction_to_tensorflow/labs/2a_dataset_api.ipynb

```diff
@@ -136,7 +136,7 @@
     "BATCH_SIZE = 3\n",
```

Comment:
<p>I had this in my list of fixes from the NR Canada cohort. You beat me to it! Thank you! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>